src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/JurisdictionServiceImpl.java

```diff
@@ -26,8 +26,8 @@
     @Autowired
     AuthTokenGenerator authTokenGenerator;
 
-    static final String errorMessage = "Jurisdiction create user profile failed or CCD service is down";
-    static final String successMessage = "Jurisdiction create user profile success!!";
+    static final String ERRORMESSAGE = "Jurisdiction create user profile failed or CCD service is down";
+    static final String SUCCESSMESSAGE = "Jurisdiction create user profile success!!";
 
```

Comment:
can you put underscore and change variables names to
ERROR_MESSAGE
SUCCESS_MESSAGE

Reply:
Sure, I will do that.

Reply:
Addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/442/commits/a46046fb012a380bc963147eba1c7f3fea65fd22